frontend/webapp/components/setup/sources/source.card/source.card.tsx

```diff
@@ -37,7 +33,10 @@ export function SourceCard({ item, onClick, focus }: any) {
             {item.name}
           </KeyvalText>
         </ApplicationNameWrapper>
-        <KeyvalTag title={item.kind} color={KIND_COLORS[item.kind]} />
+        <KeyvalTag
+          title={item.kind}
+          color={KIND_COLORS[item.kind].toLowerCase()}
```

Comment:
Current implementation will lower case the color (`"#203548".toLowerCase()`) and not the kind. 

```suggestion
          color={KIND_COLORS[item.kind.toLowerCase()]}
```